corehq/apps/geospatial/reports.py

```diff
@@ -136,6 +144,12 @@ def _build_query(self):
         query = apply_geohash_agg(query, case_property, precision)
         return query
 
+    def _get_geo_point(self, case):
+        case_obj = wrap_case_search_hit(case)
+        geo_case_property = get_geo_case_property(case_obj.domain)
+        geo_point = case_obj.case_json.get(geo_case_property)
```

Comment:
nit: good to use the standard `get_case_property` method on case.

Reply:
[6b95647](https://github.com/dimagi/commcare-hq/pull/33610/commits/6b95647ec4df4a0a9e1a2a7ab406420a7503b113)